manifests/base/runtime-rocm-pytorch-imagestream.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/runtime-image: "true"
+  annotations:
+    opendatahub.io/runtime-image-url: "https://github.com//opendatahub-io/notebooks/tree/main/runtimes"
+    opendatahub.io/runtime-image-name: "Pytorch with ROCm and Python 3.11 (UBI9)"
```

Comment:
Maybe?
```suggestion
    opendatahub.io/runtime-image-name: "Py?orch with ROCm and Python 3.11 (UBI9)"
```

Reply:
I replaced `Pytorch` with `PyTorch`; I don’t think that’s `?` what you meant :P